test/PayID/pay-id-client-test.ts

```diff
@@ -40,7 +40,10 @@ describe('Pay ID Client', function(): void {
     nock('https://xpring.money')
       .get('/georgewashington')
       .reply(200, {
-        address: 'X7cBcY4bdTTzk3LHmrKAK6GyrirkXfLHGFxzke5zTmYMfw4',
+        addressDetailsType: 'crypto',
```

Comment:
Think this should be `CryptoAddressDetails` according to @austin-king (went through it on my PR today)

Reply:
Done